circle.yml

```diff
@@ -5,6 +5,10 @@ general:
 machine:
   xcode:
     version: "7.3"
+  environment:
+    GOPATH: "$HOME/go"
+    PROJECT_ROOT: "$GOPATH/src/github.com/docker/hyperkit"
+    PATH: "${PATH}:${GOROOT}/bin:${GOPATH}/bin"
```

Comment:
It appears to be set. Come to think of it, I think setting the the `PATH` based on `GOPATH` should be sufficient. This is for it to find `golint` which should be installed in `GOPATH`